frontend/test/metabase/admin/settings/settings.cy.spec.js

```diff
@@ -0,0 +1,67 @@
+import { signInAsAdmin } from "__support__/cypress";
+
+describe("admin/settings", () => {
+  beforeEach(signInAsAdmin);
+
+  it("should save a setting", () => {
+    cy.server();
+    cy.route("PUT", "**/admin-email").as("saveSettings");
```

Comment:
@paulrosenzweig Is this pattern here used just because of the save on blur situation in settings? Are there any other situations where we'd want to use this as opposed to just interacting with the app as is?

Reply:
Yeah here it was due to the save on blur. We want to refresh and confirm that the save happened, but we can't refresh until that query completes.

Now that you bring it up, it might be needed even if there's an explicit click. Unless cypress has some unknown magic (auto wait for requests initiated during click handler?), we still can't refresh until after the PUT.

Though, in both cases, a `cy.wait` would probably be enough. Especially, if there are multiple calls that need to complete.